pubsub/gossipsub/gossipsub-v1.1.md

```diff
@@ -105,6 +105,10 @@ within the backoff period and extend it.
 When a peer tries to regraft too early, the pruning peer may apply a behavioural penalty
 for the action, and penalize the peer through P₇ (see [Peer Scoring](#peer-scoring) below).
 
+When unsubscribing from a topic, the backoff period should be finished before subscribing to
+the topic again, otherwise a healthy mesh will be difficult to reach.
+A specific backoff period for unsubscribing should be set accordingly.
```

Comment:
The UnsubBackoff is sent to peers when we unsubscribe to allow us to resubscribe before the full PruneBackoff

So while this is meant to allow fast resub, I called it the unsubbackoff because why send it during unsub

It tried to explain this better in the doc, lgty?

Reply:
lg, thanks.